BP/common/religion/religions/00_zoroastrianism.txt

```diff
@@ -468,6 +473,8 @@
 			doctrine = doctrine_spiritual_head
 		}
 		
+		#WtWSMS differences: has a HOF
```

Comment:
For a post-Islam faith?

Reply:
That's a default CK3 thing.

Reply:
Ok, but for the mod I believe it shouldn't be active at start and be without a head until an actor institutes it, after it has appeared.

Reply:
It's not active (i.e., on the map) at start. We'll need to add variables/flags to the game as well as manipulate files in common/scripted_character_templates/ to prevent pre-Islam religions from spawning; I've had a few Ashari wanderers before.

But those are for after this PR and after the map is finished (it's a feature to be implement for pre-alpha).